utils/ls_public_bucket.py

```diff
@@ -0,0 +1,217 @@
+# coding: utf-8
```

Comment:
This looks like a handy utility, but it'd be great to have a docstring here explaining what it does, and giving an example of running it.

Could be similar, but not exactly what's talked about in the notebook.

Reply:
Yes, that's right. I will add all the information on how to use it ..

Reply:
@omad said it all!